templates/zerver/create_realm.html

```diff
@@ -6,7 +6,32 @@
 {% block portico_content %}
 <script type="text/javascript">
 $(function () {
+    var timer;
+
     common.autofocus('#email');
+
+    $('#email').on("keydown", function () {
+        clearTimeout(timer);
+    });
+    $('#email').on("keyup", function () {
+        clearTimeout(timer);
+        timer = setTimeout(checkRealm, 1000, $('#email').val());
+    });
+
+    function checkRealm (email_value) {
+        $.ajax({
+          url: "/json/check_realm/" + email_value,
+          dataType: "json",
+        }).done(function (data) {
+            if (data.message === "OK") {
+                $("#message_error").removeClass("alert-error").addClass("alert-info");
+                $("#message_error").html("The organization name is valid.");
+            } else {
+                $("#message_error").removeClass("alert-info").addClass("alert-error");
+                $("#message_error").html(data['message']);
+            }
+        });
+    }
 });
```

Comment:
@elenaoat can you move this JS logic into `static/js/portico/signup.js`?  We've been trying to move direct JS out of our HTML templates.  It might make sense to start with moving the existing function to a section of `signup.js`.